packages/netsuite-adapter/src/client/client.ts

```diff
@@ -189,20 +185,19 @@ export default class NetsuiteClient {
     elemIdsAndCustInfos.forEach(elemIdAndCustInfo => {
       const currSet = dependencyMap.get(elemIdAndCustInfo.elemIdFullName)
       const endNode = dependencyGraph.findNode(elemIdAndCustInfo)
-      elemIdAndCustInfo.customizationInfos.forEach(custInfo =>
-        lookupValue(custInfo.values, val => {
-          if (!_.isString(val)) {
-            return
+      lookupValue(elemIdAndCustInfo.customizationInfo.values, val => {
+        if (!_.isString(val)) {
+          return
+        }
+        const serviceIdInfoArray = captureServiceIdInfo(val)
+        serviceIdInfoArray.forEach(serviceIdInfo => {
+          currSet.add(serviceIdInfo.serviceId)
+          const startNode = dependencyGraph.findNodeByField(SERVICE_ID, serviceIdInfo.serviceId)
```

Comment:
nit - because that is restricted to `keyof SDFObjectNode`
```suggestion
          const startNode = dependencyGraph.findNodeByField('serviceid', serviceIdInfo.serviceId)
```